component/core/src/main/java/org/exoplatform/social/core/space/SpaceUtils.java

```
@@ -1754,6 +1742,7 @@ public static boolean isUserHasMembershipTypesInGroup(String remoteId, String gr
         }
       }
     } catch (Exception e) {
+      LOG.warn("Failed to get space members, groupId = " + groupId + " and roles = " + String.join(",",membershipTypes), e);
```

ok, got it.
I still think that we should throw an exception and let the caller decide what it does in this case, but this changes is acceptable IMO.